src/main/resources/com/google/api/codegen/java/grpc_package.snip

```diff
@@ -0,0 +1,138 @@
+@snippet generate(metadata)
```

Comment:
It looks like you're copying the wrong template - you should be basing it on https://github.com/googleapis/packman/blob/master/templates/java/build-alt.gradle.mustache .